Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/decisions_tests.cy.js

```diff
@@ -0,0 +1,59 @@
+import projectTaskList from "../../pages/projectTaskList";
+import { decisionPage } from '../../pages/decisionPage';
+import { Logger } from '../../support/logger'
+
+
+
+describe('Decisions Tests', () => {
+    beforeEach(() => {
+        Logger.log("Visit the homepage before each test");
+        projectTaskList.getHomePage();
+    });
+
+    it('Creating a conversion project and recording a new decision then editting the decision', () => {
+        Logger.log("Go to the home page then click create new conversion");
+        projectTaskList.getCreateNewConversion();
+
+        Logger.log("Click on create new conversion button on the next page");
```

Comment:
This create project logic will be reusable, can you pull it out and make it sharable between modules?

Reply:
it is reusable